app/views/qa/show.html.erb

```diff
@@ -49,6 +49,11 @@
             }
           ]
         } %>
+      <% elsif single_custom_options_question? %>
```

Comment:
I don't think you need this `elsif` as the code in it is identical to the `else`.

Reply:
My bad, I was testing other ways and forgot to take it out